cypress-smoketests/fixtures/caseMigrations.js

```diff
@@ -0,0 +1,230 @@
+const faker = require('faker');
+const { MOCK_CASE } = require('../../shared/src/test/mockCase');
+
+exports.BASE_CASE = {
+  ...MOCK_CASE,
+  associatedJudge: 'Chief Judge',
+  caseCaption: 'A Migrated Casee',
+  caseId: undefined,
+  preferredTrialCity: 'Washington, District of Columbia',
+  status: 'Calendared',
+};
+
+exports.CASE_WITH_OTHER_PETITIONERS = {
```

Comment:
@ericsorenson Not a huge deal, but I see potential to DRY up this and `CASE_WITH_OTHER_FILERS` below.

Reply:
I'll add a technical card to the backlog.